cypress/integration/pages/frontPage/tests.js

```diff
@@ -6,6 +6,32 @@ const serviceHasIndexAlsos = service => service === 'thai';
 // Limiting to one service for now
 const serviceHasPublishedPromo = service => service === 'persian';
 
+// Check for valid useful links
+const shouldDisplayUsefulLinks = pageData => {
```

Comment:
What do you think about renaming this to something more clear. i.e `isValidUsefulLinks`. This communicates what the function is doing. Checking for `usefulLinks` validity.
```js
/* This gives an impression that we are displaying usefulLinks inside the if block. */
if(shouldDisplayUsefulLinks) {
}

/* This gives an impression that we want to perform some action based on the validity of usefulLinks */
if(isValidUsefulLinks) {
}
```